src/main/java/com/minecolonies/coremod/entity/citizen/EntityCitizen.java

```diff
@@ -598,7 +605,8 @@ else if (citizenData.getSaturation() < LOW_SATURATION)
             }
             else
             {
-                healAmount = 1 * (1.0 + getCitizenColonyHandler().getColony().getResearchManager().getResearchEffects().getEffectStrength(REGENERATION));;
+                healAmount = 1 * (1.0 + getCitizenColonyHandler().getColony().getResearchManager().getResearchEffects().getEffectStrength(REGENERATION));
+                ;
```

Comment:
weird format adjustment

Reply:
actually correct formatting, we just stupidly have two ";" there